test/dpg/low-level/Expected/AcceptanceTests/DPGServiceDrivenUpdateOneLowLevel/dpgservicedrivenupdateonelowlevel/rest/params/_request_builders_py3.py

```diff
@@ -203,21 +203,21 @@ def build_post_parameters_request(
 @overload
 def build_post_parameters_request(
     *,
+    content_type: str,
     content: IO,
-    content_type: Optional[str] = None,
     **kwargs: Any
 ) -> HttpRequest:
     """POST a JSON or a JPEG.
 
     See https://aka.ms/azsdk/dpcodegen/python/send_request for how to incorporate this request
     builder into your code flow.
 
+    :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
```

Comment:
Realized I didn't have this before, but we should add the content type values here. For another PR though, I might just tackle it today.